src/gui/processing/qgsprocessingmaplayercombobox.cpp

```diff
@@ -662,11 +662,15 @@ void QgsProcessingMapLayerComboBox::selectFromFile()
   setValue( filename, context );
 }
 
-void QgsProcessingMapLayerComboBox::browseForLayer()
+void QgsProcessingMapLayerComboBox::browseForVectorLayer() {
+  self.browseForLayer(QgsMapLayerType::VectorLayer)
```

Comment:
This doesn't look right -- self is a python thing, not C++ 😂

Reply:
Big oops! I was trying out things and they ended up in my commit. It should really only be a one-liner testing also for `QgsProcessingParameterVectorLayer` 🙈🙈🙈